aws-analytics-pinpoint/src/androidTest/java/com/amplifyframework/analytics/pinpoint/AnalyticsPinpointInstrumentedTest.java

```diff
@@ -71,11 +69,9 @@
     @BeforeClass
     public static void setUp() throws AmplifyException {
         Context context = getApplicationContext();
-        AmplifyConfiguration configuration =
-                AmplifyConfiguration.fromConfigFile(context, R.raw.amplifyconfiguration);
         plugin = new AmazonPinpointAnalyticsPlugin((Application) context);
         Amplify.addPlugin(plugin);
-        Amplify.configure(configuration, context);
+        Amplify.configure(context);
```

Comment:
Will it still use the configuration in the test directory?

Reply:
Yep it does. We use context to context.getPackageName() which in case of tests returns com.amplifyframework.analytics.pinpoint.test as before.